client/me/purchases/concierge-banner/index.jsx

```diff
@@ -3,39 +3,137 @@
  */
 import React, { Component } from 'react';
 import { localize } from 'i18n-calypso';
+import classnames from 'classnames';
 
 /**
  * Internal Dependencies
  */
+import Card from 'components/card';
 import ActionCard from 'components/action-card';
 import TrackComponentView from 'lib/analytics/track-component-view';
+import conciergeImage from 'assets/images/illustrations/jetpack-concierge.svg';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
 
 class ConciergeBanner extends Component {
+	placeholder() {
+		return (
+			<Card compact={ false }>
+				<div className="concierge-banner__placeholders is-placeholder">
+					<div className="concierge-banner__placeholder-row-container is-placeholder">
+						<div className="concierge-banner__placeholder-row is-placeholder" />
+						<div className="concierge-banner__placeholder-row is-placeholder" />
+						<div className="concierge-banner__placeholder-row is-placeholder" />
+					</div>
+					<div className="concierge-banner__placeholder-button-container">
+						<div className="concierge-banner__placeholder-button is-placeholder" />
+					</div>
+				</div>
+			</Card>
+		);
+	}
+
+	getText() {
+		const { bannerType } = this.props;
+
+		let headerText, mainText, buttonText, buttonHref, illustrationUrl;
+
+		switch ( bannerType ) {
+			case 'upcomingAppointment':
+				headerText = this.props.translate( 'Your appointment is coming up!' );
```

Comment:
Minor suggestion: we could destructure `translate()` out upfront so all the following `this.props` can be saved.

Theoretically speaking, this would not only save typing but also some computation cycles, since `this.props.translate` means two more JS object property lookups every time it is called. However, I'm pretty confident that any modern JS VM would optimize that out already, so it mainly saves typing only :)

Reply:
Fixed in [d8f0d66](https://github.com/Automattic/wp-calypso/pull/34956/commits/d8f0d660b9908b755844218d3ce45bda7f5ab9b0)